src/errors/Messages.js

```diff
@@ -97,6 +97,8 @@ const Messages = {
   EMOJI_MANAGED: 'Emoji is managed and has no Author.',
 
   REACTION_RESOLVE_USER: 'Couldn\'t resolve the user ID to remove from the reaction.',
+
+  VANITY_URL: 'The guild feature VANITY_URL is not enabled',
```

Comment:
This would probably be better worded with `This guild does not have the VANITY_URL feature enabled.` or something similar.